src/main/plugins/org.dita.pdf2/build_template.xml

```diff
@@ -172,8 +172,9 @@ with those set forth herein.
   </target>
 
   <target name="transform.topic2fo"
-          depends="transform.topic2fo.init,
-                   transform.topic2fo.index,
+          dita:extension="depends org.dita.dost.platform.InsertDependsAction"
+          dita:depends="transform.topic2fo.init,
+                   {depend.org.dita.pdf2.index},
```

Comment:
Moved `transform.topic2fo.index` to be used via the `depend.org.dita.pdf2.index` extension point.